test/test.js

```diff
@@ -23,4 +23,13 @@ describe("data.json", () => {
 
         assert.equal(prevArray, output.length);
     });
+
+    it.only('check if quotes have desired formats', () => {
+        data.forEach(function(item) {
+            item.quote && assert.typeOf(item.quote, 'string');
+            item.movie && assert.typeOf(item.movie, 'string');
+            item.type && assert.typeOf(item.type, 'string');
+            item.year && assert.typeOf(item.year, 'number');
+        });
+    });
```

Comment:
@Laviitz Awesome. but can you add it in another describe() so that it displays 2 test passing?
Like this.
```javascript
describe("data.json", () => {
    it("check if quotes have desired formats", () => {
        data.forEach(function(item) {
            item.quote && assert.typeOf(item.quote, "string");
            item.movie && assert.typeOf(item.movie, "string");
            item.type && assert.typeOf(item.type, "string");
            item.year && assert.typeOf(item.year, "number");
        });
    });
});
```

Reply:
I forgto to remove `it.only`, now displays 2 tests, but if you want another describe i can do it

![image](https://user-images.githubusercontent.com/37717008/66709309-82813200-ed37-11e9-9b93-3e5de4827d42.png)

Reply:
No problem, just wanted the message to display on terminal. Fine with one describe only.